docs/standard/containerized-lifecycle-architecture/what-is-docker.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: What is Docker?
-description: Containerized Docker Application Lifecycle with Microsoft Platform and Tools
+description: Get a a little deeper in your understanding of Docker, a simple analogy here might help you.
```

Comment:
This description is a bit funny.

Reply:
Sorry @mvelosop for not being clear. It's not in the sense of making me laugh, it's a bit awkward I'd say. Something like "Understand what Docker is and how it's different from virtual machines." would probably read a bit better. Makes sense?